examples/detective/invitationaccepter.yaml

```
@@ -0,0 +1,46 @@
+apiVersion: detective.aws.upbound.io/v1beta1
+kind: InvitationAccepter
+metadata:
+  annotations:
+    upjet.upbound.io/manual-intervention: "Requires AccountId1 and AccountEmail1 to replaced manually."
+  labels:
+    testing.upbound.io/example-name: member
+  name: member
+spec:
+  forProvider:
+    graphArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: primary
+    region: us-west-1
+  providerConfigRef:
+    name: test1   #Provider config for second aws account used for creation Member
+---
+apiVersion: detective.aws.upbound.io/v1beta1
+kind: Graph
+metadata:
+  annotations:
+    upjet.upbound.io/manual-intervention: "Requires AccountId1 and AccountEmail1 to replaced manually."
+  labels:
+    testing.upbound.io/example-name: primary
+  name: primary
+spec:
+  forProvider:
+    region: us-west-1
+---
+apiVersion: detective.aws.upbound.io/v1beta1
+kind: Member
+metadata:
+  annotations:
+    upjet.upbound.io/manual-intervention: "Requires AccountId1 and AccountEmail1 to replaced manually."
+  labels:
+    testing.upbound.io/example-name: primary
+  name: primary
+spec:
+  forProvider:
+    accountId: <AccountId1>
+    emailAddress: <AccountEmail1>
```

```suggestion
    accountId: ${data.aws_account_id}
    emailAddress: ${data.aws_account_root_user_email}
```

And hopefully remove manual intervention.

---

It could not be changed with accountId: ${data.aws_account_id}, emailAddress: ${data.aws_account_root_user_email}, - it is not current(root) account email, it is any another AWS user we will add as a member. For this test we user free tire user.